components/bounty/page-actions/controller.tsx

```diff
@@ -136,7 +122,7 @@ export default function PageActions({
         }) => {
         pullRequestPayload = {
             repoId,
-            issueGithubId: issueGithubID,
+            issueGithubId: currentBounty?.id,
```

Comment:
I thought we no longer had these, "issueGithuhId" makes no sense on this payload :(